src/components/App.jsx

```diff
@@ -69,9 +72,10 @@ class ContactsApp extends React.Component {
   }
 
   componentDidUpdate(prevProps) {
-    if (this.state.displayedContact) {
+    if (this.state.displayedContact && !this.state.isDisplayedContactUpdated) {
       this.setState(state => ({
         ...state,
+        isDisplayedContactUpdated: true,
         displayedContact: prevProps.contacts.find(
           contact => contact._id === state.displayedContact._id
```

Comment:
Woops, I missed this in the last review, sorry! But I think there's a problem here.

We're updating the state using `prevProps.contacts`, but in reality we want to use the current props. So I think it would be more correct to do something like this:

```
displayedContact: this.props.find(contact => contact._id === state.displayedContact._id)
```